src/shared/api/CancerHotspotsAPI.ts

```diff
@@ -0,0 +1,1044 @@
+import * as request from "superagent";
+
+type CallbackHandler = (err: any, res ? : request.Response) => void;
+export type HotspotMutation = any;
+export type SingleResidueHotspotMutation = any;
+export type IntegerRange = any;
+export type TumorTypeComposition = any;
+export type Cluster = any;
+export type ClusteredHotspotMutation = any;
```

Comment:
Hmmm.  The online client generator does seem to work for the current swagger doc, so it might be our generator.   Will have to experiment more